pkg/provisioning/validate_test.go

```diff
@@ -137,6 +137,21 @@ func TestValidator_InvalidFields(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "processor workers is negative",
+		config: PipelineConfig{
+			Status:      "running",
+			Name:        "pipeline1",
+			Description: "desc1",
+			Processors: map[string]ProcessorConfig{
+				"proc1": {
+					// invalid field
+					Type:     "js",
```

Comment:
Does this refer to `Type`?

Reply:
Oh, the comment was probably copied from another test case, I'll move it down to `Workers`.